pkg/controller/import-controller.go

```diff
@@ -582,7 +632,7 @@ func createImporterPod(log logr.Logger, client client.Client, image, verbose, pu
 // makeImporterPodSpec creates and return the importer pod spec based on the passed-in endpoint, secret and pvc.
 func makeImporterPodSpec(namespace, image, verbose, pullPolicy string, podEnvVar *importPodEnvVar, pvc *corev1.PersistentVolumeClaim, scratchPvcName *string, podResourceRequirements *corev1.ResourceRequirements) *corev1.Pod {
 	// importer pod name contains the pvc name
-	podName := importPodNameFromPvc(pvc)
+	podName := getImportPodNameFromPvc(pvc)
```

Comment:
Will the annotation always be populated at this point?  If so, why not use that value, like above?  I think it would be ideal to only call `getImportPodNameFromPvc()` once per reconcile loop

Reply:
good point

Reply:
done